tests/unit/command/test_experiments.py

```diff
@@ -418,7 +418,7 @@ def test_experiments_remove_invalid(dvc, scm, mocker, capsys, caplog):
     )
 
 
-def test_experiments_save(dvc, scm, mocker):
+def test_experiments_save(dvc, scm, mocker, caplog):
```

Comment:
```suggestion
def test_experiments_save(dvc, scm, mocker):
```

I suppose it made it here by accident?